exercises/advriskmin/ex_advriskmin_3.Rnw

```diff
@@ -0,0 +1,16 @@
+% !Rnw weave = knitr
+
+<<setup-child, include = FALSE>>=
+library('knitr')
+knitr::set_parent("../../style/preamble_ueb.Rnw")
+@
+
+\input{../../latex-math/basic-math.tex}
+\input{../../latex-math/basic-ml.tex}
+
+\kopfsl{3}
+
+\aufgabe{Risk Minimization and Gradient Descent}{
```

Comment:
Wie wäre als Titel "Connection between MLE and ERM"?

Reply:
find ich gut, habs angepasst, siehe neuer commit :)